tests/test_dicom_reader.py

```diff
@@ -10,12 +10,13 @@
 # files used from the pydicom library
 test_dcm_img_lut_file =  get_testdata_file("MR-SIEMENS-DICOM-WithOverlays.dcm")
 test_dcm_img_monochr2_file = get_testdata_file("CT_small.dcm")
+test_dcm_rgb_img = get_testdata_file("SC_rgb_jpeg.dcm")
```

Comment:
I think these test files should be chest X-ray images though. Testing on this file doesn't confirm that the code will work on a chest X-ray image that is encoded like that. I think it is better to not support it at this point.

Reply:
Nevermind! I read the commit too fast!